src/serviceWorker.js

```diff
@@ -46,7 +46,7 @@ export function register(config) {
           );
         });
       } else {
-        // Is not localhost. Just register service worker
+        // Is not localhost. Just Register service worker
         registerValidSW(swUrl, config);
       }
     });
```

Comment:
Did you really change this? Or was it generated somehow automatically?
I mean if it was made automatically, then why? Looks like something wrong added to the repo again